wait-for-solr.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Adapted from the example at: https://docs.docker.com/compose/startup-order/
+
+set -e
+
+if [[ -z ${SOLR_HOST} ]]
+then
+    # This string must match the id of the Solr service in docker-compose.yml
+    export SOLR_HOST="solr"
+fi
+
+until curl -s "http://${SOLR_HOST}:${SOLR_PORT}/solr/prl/admin/ping"; do
```

Comment:
Might be good to have a break/timeout for this (X number of times or something)? It's just going to loop forever if Solr isn't accessible?

Reply:
In that case, should there be another env var to configure that timeout value?

Reply:
or just hard-code it (since we're archiving this code anyway)?

Reply:
actually, let me know what you think of the latest commit that makes it configurable

Reply:
Looks good. I think hardcoding would have been fine too, but a configuration is even better.

I'm unsure about making the infinite loop the default, though. I think a default value (and have no opinions on what that should be) would be better. It can always be unset if an infinite loop is desired.

Reply:
I can make it behave like the third party script I was using before ([default value of 15](https://github.com/vishnubob/wait-for-it/blob/81b1373f17855a4dc21156cfe1694c31d7d1792e/wait-for-it.sh#L139), 0 means loop forever)